clusters/resource_cluster.go

```diff
@@ -47,9 +48,19 @@ func SparkConfDiffSuppressFunc(k, old, new string, d *schema.ResourceData) bool
 	return false
 }
 
+func AwsAttribsDiffSuppressFunc(k, old, new string, d *schema.ResourceData) bool {
+	if strings.HasPrefix(k, "aws_attributes.") &&
+		strings.HasSuffix(k, ".zone_id") && new == "auto" {
+		log.Printf("[DEBUG] Suppressing diff for k=%#v old=%#v new=%#v", k, old, new)
```

Comment:
Old value must be checked, otherwise this will ignore "auto" for new clusters. Very subtle, but important detail.

Reply:
done